src/Apps/CollectorProfile/Routes/Saves2/Components/Actions/__tests__/EditArtworkListModal.jest.tsx

```diff
@@ -151,7 +151,7 @@ describe("EditArtworkListModal", () => {
           __typename: "UpdateCollectionSuccess",
           artworkList: {
             internalID: "artwork-list",
-            name: "All saves",
+            name: "Saved Artworks",
```

Comment:
**suggestion**: Because this is hard-coded, maybe we use something that's obviously a placeholder like `[default]` or `[standard]`? /shrug